components/brave_ads/core/internal/creatives/conversions/creative_set_conversion_database_table.cc

```diff
@@ -160,29 +167,65 @@ void MigrateToV29(mojom::DBTransactionInfo* transaction) {
   transaction->commands.push_back(std::move(command));
 }
 
+void MigrateToV30(mojom::DBTransactionInfo* transaction) {
+  CHECK(transaction);
+
+  // Create a temporary table with a new |extract_verifiable_id| column
+  // defaulted to |true| for legacy conversions, remove the deprecated |type|
+  // column and rename the |advertiser_public_key| column to
+  // |verifiable_advertiser_public_key|.
+  mojom::DBCommandInfoPtr command = mojom::DBCommandInfo::New();
+  command->type = mojom::DBCommandInfo::Type::EXECUTE;
+  command->sql =
+      "CREATE TABLE creative_set_conversions_temp (creative_set_id TEXT NOT "
+      "NULL, url_pattern TEXT NOT NULL, extract_verifiable_id INTEGER NOT NULL "
+      "DEFAULT 1, verifiable_advertiser_public_key TEXT, observation_window "
```

Comment:
should `extract_verifiable_id` be 0 by default?

Reply:
I discussed with @iambrianfung and it was decided it was best to set this to true for legacy conversions because we did not parse the flag and assumed it was true before.